openrpc.yaml

```diff
@@ -1042,7 +1042,9 @@ components:
         iconUrls:
           description: >-
             (Optional) One or more URLs pointing to reasonably sized images that
-            can be used to visually identify the chain.
+            can be used to visually identify the chain. NOTE: MetaMask will not 
+            currently display these images. Values should still be included so 
+            that they are displayed once MetaMask displays custom iconUrls.
```

Comment:
I'm still skeptical that `iconUrls` should be part of this API but this change isn't relevant to that critique!

Reply:
Well, the signature attempts to match [EIP-3085](https://eips.ethereum.org/EIPS/eip-3085) and we just want to be transparent about whether callers should expect this parameter to be used for network display in MetaMask. I'll do another wording adjustment.